dispatches/models/fossil_case/ultra_supercritical_plant/storage/multiperiod_double_loop_usc.py

```diff
@@ -0,0 +1,369 @@
+##############################################################################
+# DISPATCHES was produced under the DOE Design Integration and Synthesis
+# Platform to Advance Tightly Coupled Hybrid Energy Systems program (DISPATCHES),
+# and is copyright (c) 2021 by the software owners: The Regents of the University
+# of California, through Lawrence Berkeley National Laboratory, National
+# Technology & Engineering Solutions of Sandia, LLC, Alliance for Sustainable
+# Energy, LLC, Battelle Energy Alliance, LLC, University of Notre Dame du Lac, et
+# al. All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. Both files are also available online at the URL:
+# "https://github.com/gmlc-dispatches/dispatches".
+#
+##############################################################################
+
+"""
+This script describes a multiperiod class to build an object for the
+integrated ultra-supercritical power plant and motlen-salt based
+thermal energy storage model.
+"""
+
+__author__ = "Naresh Susarla"
+
+import pyomo.environ as pyo
+import pandas as pd
+from collections import deque
+from dispatches.models.fossil_case.ultra_supercritical_plant \
+    .storage.multiperiod_integrated_storage_usc import create_multiperiod_usc_model
+
+
+class MultiPeriodUsc:
+    def __init__(
+        self, horizon=4, model_data=None
+    ):
+        """
+        Arguments:
+            horizon::Int64 - number of time points to use for associated multi-period model
+
+        Returns:
+            Float64: Value of power output in last time step
+        """
+        self.horizon = horizon
+        self.multiperiod_usc = None
+        self.result_list = []
+        self.result_listimp = []
+        self.model_data = model_data
+
+    def populate_model(self, b):
+        """
+        Create a integrated ultra-supercritical power plant and molten salt
+        thermal energy storage model using the `MultiPeriod` package.
+
+        Arguments:
+            blk: this is an empty block passed in from eithe a bidder or tracker
+
+        Returns:
+             None
+        """
+        tank_min = 76000  # in kg
+        tank_max = 6739292  # in kg
+
+        blk = b
+        if not blk.is_constructed():
+            blk.construct()
+
+        multiperiod_usc = create_multiperiod_usc_model(
+            n_time_points=self.horizon,
+            pmin=self.model_data.p_min, pmax=self.model_data.p_max
+        )
+        blk.usc_mp = multiperiod_usc
+        blk.rankine_model = multiperiod_usc.pyomo_model
```

Comment:
Please change the variable to usc

Reply:
The variable is not needed so it is now removed